### PR TITLE
Sosivio Predictive Troubleshooting Addon

### PR DESCRIPTION
**Pull request by DanArlowski**
### Thank you for making MicroK8s better

Please reference the issue this PR is fixing, or provide a description of the problem addressed.

*Also verify you have:*
* [x] Read the [contributions](https://github.com/ubuntu/microk8s/blob/master/CONTRIBUTING.md) page.
* [x] Submitted the [CLA form](https://ubuntu.com/legal/contributors/agreement), if you are a first time contributor.

### CODE REVIEW

File: addons/sosivio/disable [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+HELM="$SNAP/microk8s-helm3.wrapper"
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
```
Comment (neoaggelos) [inline addons/sosivio/disable:7]:
```suggestion
KUBECTL="$SNAP/microk8s-kubectl.wrapper"
```

**Comment by ktsakalozos** [comment]
Thank you @DanArlowski we will review it as soon as possible.

File: addons/sosivio/disable (hunk 1 above)
Comment (DanArlowski) [reply addons/sosivio/disable]: Fixed and squashed @neoaggelos  Thanks!

**Comment by DanArlowski** [comment]
I saw the previous run error for the CLA and linting
regarding CLA - resigned with DanArlowski (guessed its case-sensitive..)
regarding email - im pretty sure thats the email in launchpad .

regarding linting - kinda hard to understand what should be formatted without the output..

**Comment by ktsakalozos** [comment]
Thank you for the CLA.

Here are the diff reported by black:
```
jackal@aurora:~/tmp/microk8s-community-addons$ git diff
diff --git a/tests/test-addons.py b/tests/test-addons.py
index b51a654..c08ae5c 100644
--- a/tests/test-addons.py
+++ b/tests/test-addons.py
@@ -36,7 +36,7 @@ from validators import (
     validate_trivy,
     validate_argocd,
     validate_osm_edge,
-    validate_sosivio
+    validate_sosivio,
 )
 from utils import (
     microk8s_enable,
diff --git a/tests/validators.py b/tests/validators.py
index 670b105..45cfbb9 100644
--- a/tests/validators.py
+++ b/tests/validators.py
@@ -701,7 +701,10 @@ def validate_sosivio():
     Validate sosivio
     """
     wait_for_pod_state(
-        "", "sosivio", "running", label="app=sosivio-dashboard",
+        "",
+        "sosivio",
+        "running",
+        label="app=sosivio-dashboard",
         timeout_insec=300,
     )
     print("sosivio is up and running")
```

Could you please do these lint fixes? Thank you.

**Comment by ktsakalozos** [comment]
I believe there are spaces at the end of the added lines:
```
jackal@aurora:~/tmp/microk8s-community-addons$ black -v --diff  .
Identified `/home/jackal/tmp/microk8s-community-addons` as project root containing a .git directory.
Sources to be formatted: "."
.git ignored: matches the --exclude regular expression
tests/utils.py already well formatted, good job.
tests/test-addons.py already well formatted, good job.
--- tests/validators.py 2022-09-29 08:25:45.442823 +0000
+++ tests/validators.py 2022-09-29 08:27:13.664316 +0000
@@ -699,12 +699,12 @@
 def validate_sosivio():
     """
     Validate sosivio
     """
     wait_for_pod_state(
-        "", 
-        "sosivio", 
-        "running", 
+        "",
+        "sosivio",
+        "running",
         label="app=sosivio-dashboard",
         timeout_insec=300,
     )
     print("sosivio is up and running")
would reformat tests/validators.py

All done! ✨ 🍰 ✨
1 file would be reformatted, 2 files would be left unchanged.
```

**Comment by ktsakalozos** [comment]
@DanArlowski You can also click on the checkbox to allow the maintainers to edit your PR and fix this kind of minor issues :)

**Comment by DanArlowski** [comment]
@ktsakalozos - pushed the fix

FYI - I dont see any tick box - probably because its an organization fork and not a private one - so its an extra precaution from github's side.

Thanks!

**Pull request merged**